applications/nrf_desktop/src/modules/ble_scan.c

```diff
@@ -116,12 +131,29 @@ static int configure_filters(void)
 		}
 		filter_mode |= BT_SCAN_ADDR_FILTER;
 	} else {
-		LOG_INF("Device type filters added");
+		u8_t connected_peers_mask = 0;
```

Comment:
maybe this could be shorter so line 149 would fit

Reply:
done